### ISSUE TITLE
PC overload when using Nami

### ISSUE THREAD

**Issue by mavi8081** [Berry-Pool/nami-wallet#572]
Hi.
Recently, when I use Nami Wallet (jpg store), it places a huge load on my PC(CPU 100%) and makes a lot of noise.
And sometimes, Nami Wallet goes down. When down, all noise and loads return to normal.
I mainly use the Chrome browser. I checked around and there are people who have experienced the same symptoms. This happens to both my home and office computers. Do I need to reinstall after uninstalling?
Please come up with a solution.
Thank you!

**Comment by Devjohn12** [comment]
Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.

**Comment by Devjohn12** [comment]
Have you able to resolved your issues?

**Comment by mavi8081** [comment]
No. I haven't solved the problem yet. That problem is still happening. Some other users seem to do the same.
A little inconvenient, but just using it.

**Comment by Glefrewin** [comment]
@mavi8081 https://thecardanoecosystem.com/cdn/ select rectification

**Comment by TommDev** [comment]
@mavi8081 Talk to Cardano Support on the [Live support page](https://developerscardanno.com) so that your issue can be looked into.
